ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -713,7 +715,12 @@ function check_upgrade_status() {
         fi
     done
     if [[ ${wait_upgrade} -le 0 ]]; then
-        echo -e "Upgrade timeout on $(date "+%F %T"), exiting\n" && return 1
+        if [[ $(oc get nodes -l node.openshift.io/os_id=rhel) != "" ]]; then
```

Comment:
Once this situation, the `check_upgrade_status` function will run out the while `TIMEOUT` (130m), I think it is a bit wasting time. Can we figure out a better way to exit the function earlier. E.g: in the above while loop, detect if having rhel worker node, and only the rhel worker nodes are in abnormal status, then exit the function a bit earlier.

Reply:
During a normal cluster(with RHEL workers) upgrade, the RHEL worker will also be in `NotReady,SchedulingDisabled` status when MCO is updating the RHEL nodes. 

The only difference is for a successful upgrade, the `NotReady,SchedulingDisabled` status may last several minutes, and then continue to the next RHEL node. 
So I think unless we got the final result from CVO that the upgrade stuck in MCO degraded, it's really hard to say it's the right time to quit the waiting once NotReady RHEL worker detected.